src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -222,12 +225,41 @@ export default {
 		};
 	},
 	apollo: {
-		query: userIdQuery,
-		preFetch: true,
-		result({ data }) {
-			this.userId = data?.my?.userAccount?.id ?? null;
+		preFetch(config, client) {
+			return client.query({
+				query: userIdQuery
+			}).then(() => {
+				return client.query({
+					query: itemsInBasketQuery
+				});
+			});
 		},
 	},
+	created() {
+		const basketId = this.cookieStore.get('kvbskt');
+
+		try {
+			const userIdData = this.apollo.readQuery({
+				query: userIdQuery
```

Comment:
I haven't dug into this usage of `apollo` yet, I'm assuming this will pull from the cache filled by `preFetch`?

Reply:
Correct. read the same query that pushed data to the cache to get that data outside of the preFetch method.